helm/ph-ee-engine/requirements.yaml

```diff
@@ -96,10 +96,10 @@ dependencies:
     version: 1.0.0
     repository: "file://./kafka"
     condition: "kafka.enabled"
-  - name: connector_bulk_closedloop
+  - name: connector
```

Comment:
this can be bulk connector and existing one should be bulk processor?

Reply:
This will be addressed later as backlog